hbase-checkstyle/src/main/resources/hbase/checkstyle-suppressions.xml

```diff
@@ -36,13 +36,13 @@
   <suppress checks="MagicNumberCheck" files=".*/src/test/.*\.java"/>
   <suppress checks="VisibilityModifier" files=".*/src/test/.*\.java"/>
   <suppress checks="InterfaceIsTypeCheck" files=".*/src/main/.*\.java"/>
-  <suppress checks="EmptyBlockCheck" files="TBoundedThreadPoolServer.java"/>
-  <suppress checks="EmptyBlockCheck" files="TestServletFilter.java"/>
-  <suppress checks="EmptyBlockCheck" files="TestGlobalFilter.java"/>
-  <suppress checks="EmptyBlockCheck" files="TestPathFilter.java"/>
-  <suppress checks="EqualsHashCode" files="StartcodeAgnosticServerName.java"/>
-  <suppress checks="MethodLength" files="Branch1CoprocessorMethods.java"/>
+  <suppress checks="EmptyBlockCheck" files="org.apache.hadoop.hbase.thrift.TBoundedThreadPoolServer.java"/>
+  <suppress checks="EmptyBlockCheck" files="org.apache.hadoop.hbase.http.TestServletFilter.java"/>
+  <suppress checks="EmptyBlockCheck" files="org.apache.hadoop.hbase.http.TestGlobalFilter.java"/>
+  <suppress checks="EmptyBlockCheck" files="org.apache.hadoop.hbase.http.TestPathFilter.java"/>
+  <suppress checks="EqualsHashCode" files="org.apache.hadoop.hbase.favored.StartcodeAgnosticServerName.java"/>
```

Comment:
Unrelated to this jira, is the naming of "StartcodeAgnosticServerName" class correct? Like, is it intentionally not StartCodeAgnosticServerName?

Reply:
Not sure if is correct, but it is consistent. This class relates to `ServerName.NON_STARTCODE`, which should be `ServerName.NON_START_CODE` if the name changes.